PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+If you’re adding a new scraper, please ensure that you have:
+
+- [ ] Tested the scraper on a local copy of DevDocs
+- [ ] Ensured that the docs are styled similarly to other docs on DevDocs
+- [ ] Added these files to the <code>public/icons/*your_scraper_name*/</code> directory:
```

Comment:
Not all documentations have a custom icon (see JSDoc and Koa for example), so I think it's a good idea to change this into an "If possible, do ..." kind of task.